contracts/erc1155/contracts/src/ERC1155Mintable.sol

```diff
@@ -63,6 +63,34 @@ contract ERC1155Mintable is
         }
     }
 
+    /// @dev creates a new token
+    /// @param type_ of token
+    /// @param uri URI of token
+    /// @param isNF is non-fungible token
+    function createWithType(
+        uint256 type_,
+        string calldata uri,
+        bool isNF
```

Comment:
Don't see this parameter being used. Placeholder?

Reply:
Ah, yeah. Taken from regular `create` but isn't needed. I'll take it out.